tests/unit/browser/test_cookies.py

```diff
@@ -0,0 +1,149 @@
+# vim: ft=python fileencoding=utf-8 sts=4 sw=4 et:
+
+# Copyright 2015 Alexander Cogneau (acogneau) <alexander.cogneau@gmail.com>:
+#
+# This file is part of qutebrowser.
+#
+# qutebrowser is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# qutebrowser is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with qutebrowser.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Tests for qutebrowser.browser.cookies"""
+
+from unittest import mock
+
+from PyQt5.QtNetwork import QNetworkCookie
+from PyQt5.QtTest import QSignalSpy
+from PyQt5.QtCore import QUrl
+import pytest
+
+from qutebrowser.browser import cookies
+from qutebrowser.utils import objreg
+from qutebrowser.misc import lineparser
+
+CONFIG_ALL_COOKIES = {'content': {'cookies-accept': 'all'}}
+CONFIG_NEVER_COOKIES = {'content': {'cookies-accept': 'never'}}
+CONFIG_COOKIES_ENABLED = {'content': {'cookies-store': True}}
+
+
+cookie1 = b'foo1=bar; expires=Tue, 01-Jan-2036 08:00:01 GMT'
+cookie2 = b'foo2=bar; expires=Tue, 01-Jan-2036 08:00:01 GMT'
+session_cookie = b'foo3=bar'
+expired_cookie = b'foo4=bar; expires=Sat, 01-Jan-2000 08:00:01 GMT'
+
+
+class LineparserSaveStub(lineparser.LineParser):
```

Comment:
You should only inherit `BaseLineParser` here, and provide stubs for the stuff needed. Otherwise, this will probably read/save test data from/to disk.